src/windows/functions.cpp

```diff
@@ -393,6 +393,37 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterEjectable(TCHAR driveLetter) {
```

Comment:
One – I suspect easier – way to do it would maybe be to expose this through the node bindings, and unit test in JS with platform checks in the test suites;
```js
describe('c++ thing on windows', function() {

  if(os.platform() !== 'win32') {
    this.skip()
  }

  it('...', function() {...})

})
```